website/docs/index.html.markdown

```diff
@@ -34,12 +34,11 @@ resource "github_membership" "membership_for_user_x" {
 
 The following arguments are supported in the `provider` block:
 
-* `token` - (Optional) This is the GitHub personal access token. It must be provided, but
+* `token` - (Required) This is the GitHub personal access token. It must be provided, but
   it can also be sourced from the `GITHUB_TOKEN` environment variable.
 
-* `organization` - (Optional) This is the target GitHub organization to manage. The account
-  corresponding to the token will need "owner" privileges for this organization. It must be provided, but
-  it can also be sourced from the `GITHUB_ORGANIZATION` environment variable.
+* `organization` - (Required) This is the target GitHub organization to manage. The account
```

Comment:
technically both of these are still optional since they can be sourced from the Environment Variables?

Reply:
Yeah I went to look at how AWS does it, they are inconsistent in their usage of it as well: https://www.terraform.io/docs/providers/aws/#region

IMO it should be marked as `Required` since it is a required piece of data, but the env vars list other ways to pass it, but maybe you feel strongly the other way?

Reply:
Perhaps it'd be clearer as `(Required unless specified through the environment variable `FOO_BAR`)?